content/en/blog/2020/deploy-wasm-declarative/index.md

```diff
@@ -0,0 +1,154 @@
+    alt="How the wasme operator works"
+    caption="Understanding how wasme operator works"
+    >}}
+
+At the moment, the Wasm image needs to be published into a registry for the operator to correctly cache it. The cache pods run as DaemonSet on each node so that the cache can be mounted into the Envoy container. This is being improved, as it's not the ideal mechanism, so stay tuned for updates. The mount is established by using the `sidecar.istio.io/userVolume` and `sidecar.istio.io/userVolumeMount` annotations. See [the docs on Istio Resource Annotations](/docs/reference/config/annotations/) for more about how that works. 
+
+Once the Wasm module is cached correctly and mounted into the workload's service proxy, the operator then configures the `EnvoyFilter` resources. 
+
+```yaml
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
+metadata:
+  name: details-v1-myfilter
+  namespace: bookinfo
+spec:
+  configPatches:
+  - applyTo: HTTP_FILTER
+    match:
+      context: SIDECAR_INBOUND
+      listener:
+        filterChain:
+          filter:
+            name: envoy.http_connection_manager
+            subFilter:
+              name: envoy.router
+    patch:
+      operation: INSERT_BEFORE
+      value:
+        config:
+          config:
+            configuration: tomorrow
+            name: myfilter
+            rootId: add_header
+            vmConfig:
+              code:
+                local:
+                  filename: /var/local/lib/wasme-cache/44bf95b368e78fafb663020b43cf099b23fc6032814653f2f47e4d20643e7267
+              runtime: envoy.wasm.runtime.v8
+              vmId: myfilter
+        name: envoy.filters.http.wasm
+  workloadSelector:
+    labels:
+      app: details
+      version: v1
+```
+
+You can see the `EnvoyFilter` resource configures the proxy to add the `envoy.filter.http.wasm` filter and load the Wasm module from the `wasme-cache`. 
+
+Once the Wasm extension is loaded into the Istio service proxy, it will extend the capabilities of the proxy with whatever custom code you introduced. 
+
+## Next Steps
+
+In this blog we explored options for installing Wasm extensions into Istio workloads. The easiest way to get started with WebAssembly on Istio is to use the `wasme` tool [to bootstrap a new Wasm project](https://docs.solo.io/web-assembly-hub/latest/tutorial_code/getting_started/) with C++, AssemblyScript [or Rust coming really soon!]. For example, to set up a C++ Wasm module, you can run:
+
+```
+wasme init ./filter --language cpp --platform istio --platform-version 1.5.x
+```
+
+If we didn't have the extra flags, `wasme init` would enter an interactive mode walking you through the correct values to choose. 
+
+Take a look at the [WebAssembly Hub wasme tooling](https://docs.solo.io/web-assembly-hub/latest/tutorial_code/getting_started/) to get started with Wasm on Istio.
+
+## Learn more
+
+-   Redefine Extensibility [with WebAssembly on Envoy and Istio](/blog/2020/wasm-announce/)
```

Comment:
Need to use a consistent style for list items. Since earlier you used `*`, you need to either change those to `-` or change all of the following ones to `*`.